docs/guides/ruby-quick-ref/index.md

```diff
@@ -616,10 +616,13 @@ end</pre>
 
 <div class="level">
   <div class="level-left">
+    <a class="button" href="../using-ronin-db/">
+      &laquo; Using Ronin DB
+    </a>
```

Comment:
It doesn't make sense to link from the "Ruby Quick Ref" guide which is about scripting and writing Ruby, back to the "Using the Ronin DB" guide which is about using the `ronin-db` command. Also, any additions should be submitted in a separate PR.